src/infra/alarm/template.yaml

```
@@ -0,0 +1,60 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: AWS::Serverless-2016-10-31
+
+Description: >-
+  Account Interventions Service's Alarm stack for ID Reuse.
+
+Parameters:
+  CoreStackName:
+    Description: The name of the Account Interventions Service's Stack
```

```suggestion
    Description: The name of the Core AIS Stack
```

To state this is the Core stack reference